lib/repetitivo/functions/strip-units.scss

```diff
@@ -1,8 +1,8 @@
-// Strip units from a number
-// @param {Number} $number The number to strip
-// @return {Number}
-// @example
-//   strip-units(4px) // 4
+/// Strip units from a number
+/// @param {number} $number The number to strip
```

Comment:
There should be a dash between `$number` and `The number to strip`, right?

Reply:
Yes, good catch!